### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by Trictonicmp**
#### In this milestone I:
- Setup a react project
- Added dependency react-router-dom to be able to change pages
- Added navbar component
- Added navbar styles
- Added booklist component
- Added styles to booklist component
- Added book component
- Added styles to book component
- Added addbookform component
- Added styles to addbookform component

### CODE REVIEW

File: src/components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const Nav = () => (
+  <header className={styles.navbar}>
+    <div className={`page-container ${styles.pageContainer}`}>
+      <Link to="/books" className={styles.logo}>
+        Bookstore CMS
+      </Link>
+      <ul className={styles.linksContainer}>
+        <li><Link to="/books" className={`${styles.link} ${styles.activeLink}`}>BOOKS</Link></li>
+        <li><Link to="/categories" className={styles.link}>CATEGORIES</Link></li>
+      </ul>
```
Comment (roshan-bajgain) [inline src/components/Nav.jsx]: - [x] There should be 2 routes books and Categories and when we click books it should `renders a form with inputs for the title and the author of the book`, `Has a button for removing books, just a place holder`, `Has a button to submit`  the button wouldn't add the book *yet*, this is just a placeholder for now.
- [x] When we click the `Categories` Should display a button "Check status" only. But is showing empty when I render categories. Kindly fix it.
- [x] You have good styling the only thing is you need to make the book's routes a default route when I click books links, it is empty at this point. eg:
 `<Route exact path="/">
      <BookList />
    </Route>`

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Trictonicmp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Nav.jsx (hunk 1 above)
Comment (Trictonicmp) [reply src/components/Nav.jsx:14]: Hi @roshan-bajgain, thank you for the feedback but I have a question, where should the button to remove books go? in the form? or on each book component?

Comment (roshan-bajgain) [reply src/components/Nav.jsx]: @Trictonicmp Inside books routes there should be an input field for title and author and make some hardcoded for author and title and each has a removing buttons. Kindly implement it

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Trictonicmp,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

# Highlight
✅ No linter error
✅ README is professional
✅ UI is starting up with good looks. 
✅ Good commit messages
✅ Good descriptive PR

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**